### ISSUE TITLE
Issue

### ISSUE THREAD

**Issue by russell2022** [jurialmunkey/skin.eminence.2#112]
Bluetooth game controller stops working when changing to this skin.

**Comment by jurialmunkey** [comment]
Highly unlikely to be a skin issue. Skins don't modify keymaps or have any interaction with bluetooth

**Comment by russell2022** [comment]
I would agree but the fact remains any of my working Bluetooth gamepads stop working with this skin. I must change/reboot to default (Estuary) skin for them to work. Many of my friends have noticed the same issue.

**Comment by russell2022** [comment]
Thanks in advance for your help with this issue. It's an excellent skin and
I've/we've been using it for years. Hopefully you can determine what's
causing the issue. If you have a Bluetooth or 2.4 wireless gamepad, try it
and I would assume the issue will surface for you to investigate.
